observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6"]
```

Comment:
```suggestion
[id="network-observability-operator-release-notes-1-6_{context}"]
```
Don't know if you want to add the context tag to your headings or not; OCP is doing it, but I haven't decided for MicroShift, either. Food for thought.

Reply:
Ugh, I know. I saw that come in team-ocp-docs and I was like man, I _guess_ I'll add it but I haven't done it yet because I already had all these titles at the time that memo came through. Kinda bad timing with OCP release.